js/ol3-sidebar.js

```diff
@@ -1,3 +1,10 @@
+// detect jsdom environment
+// borrowed from: https://github.com/jsdom/jsdom/issues/1537
+var ol;
+if (navigator.userAgent.includes("Node.js") || navigator.userAgent.includes("jsdom")) {
+  ol = require('openlayers');
+}
```

Comment:
since this is needed only for tests I would prefer it if we could do this in the test file instead 🤔

Reply:
If I could have left the main code alone, I would have.  I certainly tried very hard! Unfortunately, there seems to be no way to have the sidebar code "know" about OpenLayers (i.e. the `ol` namespace) at `require` time other than explicitly `require`-ing it inside the module itself. It didn't seem possible to load a `<script>` within the HTML loaded as part of jsdom in the jest setup, so it didn't seem to be possible to put `ol` into the global namespace for sidebar to find it. Do you have any ideas as to how one might be able to get around this?